packages/eslint-plugin/src/rules/index.ts

```diff
@@ -16,6 +16,7 @@ import * as redundantUndefined from "./redundant-undefined";
 import * as trimFile from "./trim-file";
 
 export const rules = {
+  "@typescript-eslint/prefer-namespace-keyword": "error",
```

Comment:
Did you mean to stick this into the preset file? This is the rules object for the plugin.

Reply:
Oh, yes, thank you